src/Symfony/Component/Form/Command/FormDebugCommand.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Form\FormRegistryInterface;
+use Symfony\Component\Form\ResolvedFormTypeInterface;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A console command for retrieving information about form types.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class FormDebugCommand extends Command
+{
+    private $formRegistry;
+    private $optionsResolver;
+    private $parents = array();
+    private $extensions = array();
+    private $namespaces;
+
+    public function __construct(FormRegistryInterface $formRegistry, array $namespaces = array('Symfony\\Component\\Form\\Extension\\Core\\Type'))
```

Comment:
See https://github.com/symfony/symfony/pull/23694#discussion_r130217875

The reasoning behind is that it's only a list of "preferred" namespaces, with a sensible default if the command is instantiated without.
If there are more namespaces to register, you still have to pass the full list including the core one, because you're likely to automate this like we did in the `FormPass`, and using `array_unique` here should be avoided IMHO (by principe I avoid to transform injected arguments unless there is very good reason to do so).
In conclusion, let's keep everything exactly as configured by the developper, no operation behind his back.